robocop/checkers/misc.py

```diff
@@ -1939,3 +1949,30 @@ def _report(self, rule_name: str, node):
             col=node.col_offset + 1,
             end_col=node.col_offset + len(node.value) + 1,
         )
+
+
+class UndefinedArgumentDefaultChecker(VisitorChecker):
+    reports = ("undefined-argument-default",)
+
+    def visit_Arguments(self, node: Arguments):  # noqa: N802
+        for token in node.get_tokens(Token.ARGUMENT):
+            arg = token.value
+
+            # From the Robot User Guide:
+            # "The syntax for default values is space sensitive. Spaces before
+            # the `=` sign are not allowed."
+            if "}=" not in arg:
+                # has no default
+                continue
+
+            (arg_name, default_val) = arg.split("}=", maxsplit=1)
```

Comment:
() is not required, by default everything with , is considered 'tuple' in Python and works with unpacking etc

```suggestion
            arg_name, default_val = arg.split("}=", maxsplit=1)
```